frontend/app/module/input/page/FinaliseElectionPage.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { RouterProvider } from "react-router-dom";
+
+import { render as rtlRender, screen } from "@testing-library/react";
+import { describe, expect, test, vi } from "vitest";
+
+import { expectNotFound, overrideOnce, render, setupTestRouter } from "app/test/unit";
+
+import { ApiProvider, ElectionProvider, ElectionStatusProvider } from "@kiesraad/api";
+
+import { FinaliseElectionPage } from "./FinaliseElectionPage";
+
+describe("FinaliseElectionPage", () => {
+  test("Error when election is not ready", async () => {
+    // Since we test what happens after an error, we want vitest to ignore them
+    vi.spyOn(console, "error").mockImplementation(() => {
+      /* do nothing */
+    });
+    const router = setupTestRouter();
+
+    await router.navigate("/1/input/finalise");
+
+    // NOTE: We're not using the wrapped render function here,
+    // since we want control over our own memory router.
+    rtlRender(
+      <ApiProvider host="http://testhost">
+        <RouterProvider router={router} />
+      </ApiProvider>,
+    );
+
+    await expectNotFound();
+  });
+});
+
+describe("FinaliseElectionPage", () => {
```

Comment:
I don't think this is `describe` is needed here, both tests can live in the same one.

Reply:
Fixed in [bab01a5](https://github.com/kiesraad/abacus/pull/274/commits/bab01a541edf3441b06faef95e7b6f6b371b8a99)